content/posts/vue3.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Vue3 upgrade"
+date: 2023-09-24
+version: ["hint v3.0.0"]
```

Comment:
This is the same version as the Help videos, but nominal publication date is different.

Reply:
Oh yeah I just guessed at when we deployed vue3.. that was a bit ago? Didn't think it was actually very important to get that spot on. Help is v3.3.0

Reply:
No, it probably doesn't matter too much